wicket-core/src/main/java/org/apache/wicket/ajax/StatelessOnChangeAjaxBehavior.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.wicket.ajax;
+
+import org.apache.wicket.Component;
+import org.apache.wicket.ajax.attributes.AjaxRequestAttributes;
+import org.apache.wicket.ajax.form.OnChangeAjaxBehavior;
+import org.apache.wicket.ajax.form.StatelessAjaxFormComponentUpdatingBehavior;
+import org.apache.wicket.markup.html.form.TextArea;
+import org.apache.wicket.markup.html.form.TextField;
+
+/**
+ * Stateless version of {@link OnChangeAjaxBehavior}.
+ * 
+ * <p>
+ * This behavior uses best available method to track changes on different types of form components.
+ * To accomplish this for text input form components it uses a custom event, named 'inputchange',
+ * that is handled in the best way for the specific browser. For other form component types the
+ * 'change' event is used.
+ * </p>
+ * 
+ * @author Andrea Del Bene
+ *
+ */
+public class StatelessOnChangeAjaxBehavior extends StatelessAjaxFormComponentUpdatingBehavior
```

Comment:
Except the parent class this class is an exact copy of `OnChangeAjaxBehavior`.
Can't we extract the common code somewhere and reuse it ?